### ISSUE TITLE
Error converting value "IEX" to type 'Alpaca.Markets.Exchange'. Path '[7335].exchange', line 1, position 1410521.

### ISSUE THREAD

**Issue by nkoehler** [alpacahq/alpaca-trade-api-csharp#134]
This was working 3 days ago, and today I ran it and it resulted in this error.  Looks like a problem with enum mapping.

ArgumentException: Requested value 'IEX' was not found.

Error converting value "IEX" to type 'Alpaca.Markets.Exchange'. Path '[7335].exchange', line 1, position 1410521.

Code:
```csharp
var assets = await restClient.ListAssetsAsync(AssetStatus.Active, AssetClass.UsEquity);
```

StackTrace:
```csharp
at Newtonsoft.Json.Converters.StringEnumConverter.ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.DeserializeConvertable(JsonConverter converter, JsonReader reader, Type objectType, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonSerializer.Deserialize[T](JsonReader reader)
   at Alpaca.Markets.RestClient.deserializeAsync[TApi,TJson](HttpResponseMessage response)
   at Alpaca.Markets.RestClient.callAndDeserializeSingleObjectAsync[TApi,TJson](HttpClient httpClient, IThrottler throttler, Uri endpointUri, CancellationToken cancellationToken, HttpMethod method)
   at Alpaca.Markets.RestClient.getObjectsListAsync[TApi,TJson](HttpClient httpClient, IThrottler throttler, UriBuilder uriBuilder, CancellationToken cancellationToken)
```
Any help is appreciated, thanks.

**Comment by OlegRa** [comment]
@nkoehler I know root cause of this problem and I'll try to fix it ASAP.

**Comment by OlegRa** [comment]
@nkoehler Can be teste in version [3.3.3](https://www.nuget.org/packages/Alpaca.Markets/3.3.3) available on NuGet.

**Comment by nkoehler** [comment]
The issue is fixed in version 3.3.3.  Thank you for responding and fixing it so quickly! I truly appreciate the effort!
